packages/ember-runtime/lib/system/core_object.js

```diff
@@ -523,20 +523,20 @@ var ClassMixinProps = {
     Creates a new subclass.
 
     ```javascript
-    App.Person = Ember.Object.extend({
-      say: function(thing) {
+    const Person = Ember.Object.extend({
+      say(thing) {
         alert(thing);
        }
     });
     ```
 
-    This defines a new subclass of Ember.Object: `App.Person`. It contains one method: `say()`.
+    This defines a new subclass of Ember.Object: `Person`. It contains one method: `say()`.
 
     You can also create a subclass from any existing class by calling its `extend()` method.
-    For example, you might want to create a subclass of Ember's built-in `Ember.View` class:
+    For example, you might want to create a subclass of Ember's built-in `Ember.Component` class:
 
     ```javascript
-    App.PersonView = Ember.View.extend({
+    App.PersonComponent = Ember.Component.extend({
```

Comment:
I'd prefer to use `export default` here, but I assume you thought of that and discarded it when updating this.  Can you share your thoughts here?